languages/elixir/exercises/concept/protocols/.docs/after.md

```diff
@@ -0,0 +1,82 @@
+Protocols are:
+
+- a mechanism to achieve polymorphism.
+- defined with `defprotocol`[defprotocol].
+- implemented for a specific data type with `defimpl`[defimpl].
+
+A big strength of protocols is that the implementation of a protocol for a specific data type doesn't need to be bundled with the data type's definition or the protocol's definition. This allows for writing libraries that can be easily extended by their users to work with the users' custom data types, without having to modify the library.
+
+## Implementing
+
+It is possible to implement protocols for all Elixir types: structs, `Tuple`, `Atom`, `List`, `BitString`, `Integer`, `Float`, `Function`, `PID`, `Map`, `Port`, `Reference`, and `Any`.
```

Comment:
Should structs be within backquotes too?

Reply:
My thinking is that it shouldn't because it differs from all the other items in that they can be substituted for Y in `defimpl X, for: Y do`, but for structs you need to use the actual name of a struct.

Reply:
I agree with the above reasoning. A C# example: "classes" should not be in backticks (it is not an actual keyword), but `class` should be.